README.md

```diff
@@ -7,7 +7,7 @@ There is an [open issue](https://github.com/helm/helm/issues/2247) and [PR](http
 ## Install
 
 ```bash
-wget -qO- "https://github.com/signavio/k8s-helm-dep-updater/releases/latest/download/k8s-helm-dep-updater_$(uname -s)_$(uname -m).tar.gz" | tar -C /tmp -xzf- k8s-helm-dep-updater
+wget -qO- "https://github.com/signavio/k8s-helm-dep-updater/releases/download/v1.1.1/k8s-helm-dep-updater_$(uname -s)_$(uname -m).tar.gz" | tar -C /tmp -xzf- k8s-helm-dep-updater
```

Comment:
In the readme we should not pin the version. We can use latest, but modify the install script so it covers the `amd64` option. right now `uname -m` will print out architecture of the cpu for example `arm64`, `x86_64` or `i686`, please find command that will print out the new value

Reply:
Since the last version may have a different name from v1.1.1 of the package, I changed the URL. Also, I believe we will start using a specific version of the package from now on, instead of the latest one, to better keep track and simplify debugging. Regarding uname -m, AMD64 refers to the 64-bit version that should run on modern Intel/AMD 'PC' hardware, as well as later Apple models. Hence, the result can vary depending on the hardware we use. Do you have specific hardware in mind when you want to run this bash script and obtain a different result?